### PR TITLE
refactor(crypto): change MaximumPaymentCountExceededError error

### LINKED ISSUE #3133: [2.6][T4] MaximumPaymentCountExceededError shows incorrect received amount
When creating a multipayment with more than 500 payments, the error message will always show a received amount of '501', even though the number is greater than 501.

```
UnhandledPromiseRejectionWarning: MaximumPaymentCountExceededError: Expected a maximum of 500 payments, but got 501.
```

**Expected Behavior**
Show the correct amount of received payments or amend the error message to ommit the received amount.

**Your Environment**
Devnet 2.6

### PR DESCRIPTION
**Pull request by Lemii**
## Summary
Closes #3133 .

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [X] Ready to be merged

### CODE REVIEW

File: packages/crypto/src/errors.ts [hunk 1]
```diff
@@ -127,8 +127,8 @@ export class MissingMilestoneFeeError extends CryptoError {
 }
 
 export class MaximumPaymentCountExceededError extends CryptoError {
-    constructor(given: number) {
-        super(`Expected a maximum of 500 payments, but got ${given}.`);
+    constructor(limit: number) {
+        super(`Number of payments exceed the expected maximum of ${limit}.`);
```
Comment (faustbrian) [inline packages/crypto/src/errors.ts]:
```suggestion
        super(`Number of payments exceeded the allowed maximum of ${limit}.`);
```

**Comment by faustbrian** [comment]
@Lemii 1 unit test is failing.

**Comment by dated** [comment]
With these changes the error is triggered only after `limit + 1` transactions have been successfully added already.

**Comment by Lemii** [comment]
You're right. Will change it back. Apologies for the noise.

**Pull request merged**